# Generated for NaoQi method ALRedBallTracker.stopTracker
# Start the tracker by Unsubscribing to Event redBallDetected from ALRedBallDetection module.

---


